NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs

```diff
@@ -23,6 +24,7 @@ public AspNetSessionValueLayoutRendererTests()
 
         public void SetUp()
         {
+            ConfigurationItemFactory.Default.LayoutRenderers.RegisterDefinition("aspnet-session", typeof(AspNetSessionValueLayoutRenderer));
```

Comment:
tested it locally VS2013 update 5+R# 10.0.2: no problem;

Will try VS2015 also